### ISSUE TITLE
Clementine not grabbing GTK Theme in Ubuntu 20.04 running Gnome 3.36.2

### ISSUE THREAD

**Issue by BeholdenCypress** [clementine-player/Clementine#6689]
### Before posting

Please follow the steps below and check the boxes with [x] once you did the step.

- [x] I checked the issue tracker for similar issues
- [x] I checked the [changelog](https://github.com/clementine-player/Clementine/blob/master/Changelog) if the issue is already resolved
- [x] I tried the latest Clementine build from [here](https://builds.clementine-player.org/)

### System information

Please provide information about your system and the version of Clementine used.

- **Operating System**: Ubuntu 20.04 LTS
- **Desktop Environment**: Gnome 3.36.2
- **Clementine version**: Version 1.4.0rc1-231-g60a46d193

### Expected behaviour / actual behaviour

- **Expected Behavior**: Clementine should be grabbing the theme (Arc-Dark) from the system.
- **Actual Behavior**: Clementine is using the default (ugly) white theme when _**Use the system default color set**_ is selected under Perferences > Appearance.

[https://i.imgur.com/HwRb7sK.png](url)

**Comment by dmdmdm** [comment]
Hi. Clementine uses QT.  I haven't tried this in your exact environment but it sounds like it might work:
https://askubuntu.com/questions/910012/how-can-i-get-qt5-applications-to-use-the-gtk-theme-in-ubuntu-17-04

**Comment by funwhilelost** [comment]
@BeholdenCypress I was able to get a pretty close approximation on Ubuntu 20.04 by using [Kvantum from this tutorial](https://www.linuxuprising.com/2018/05/use-custom-themes-for-qt-applications.html).

tl;dr
```
sudo apt install qt5-style-kvantum qt5-style-kvantum-themes
echo "export QT_STYLE_OVERRIDE=kvantum" >> ~/.profile
```
Log out, log in, run "Kvantum Manager" from activities and you should be good to go!

![image](https://user-images.githubusercontent.com/127476/98032477-ee15c180-1dc8-11eb-91d5-a5c7b7e2d452.png)
